test_unstructured/partition/test_api.py

```diff
@@ -73,34 +73,6 @@ def test_partition_via_api_from_filename(monkeypatch):
     assert elements[0].metadata.filetype == "message/rfc822"
 
 
-def test_partition_via_api_custom_url(monkeypatch):
-    """
-    Assert that we can specify api_url and requests are sent to the right place
-    """
```

Comment:
It seems to me this test should be in the SDK repos instead since it just passes `base_url` through to the SDK. This test is failing because it gets stuck in retries before failing.

Reply:
Yeah that makes sense to me. You could probably update the mock response to prevent the retries, but I don't think the effort's worth and, like you said, test makes more sense on the SDK side.